test/v3/connectors/TriggerBsasConnectorSpec.scala

```diff
@@ -47,5 +48,20 @@ class TriggerBsasConnectorSpec extends ConnectorSpec {
 
       await(connector.triggerBsas(request)) shouldBe outcome
     }
+
+    "post a TriggerBsasRequest body and return the result given a TYS tax year" in new TysIfsTest with Test {
+      val request = TriggerBsasRequest(nino, tysModel)
+      val outcome = Right(ResponseWrapper(correlationId, TriggerBsasResponse(calculationId)))
+
+      val taxYear: TaxYear = TaxYear.fromIso("2023-05-02")
+
+      willPost(
+        url = s"$baseUrl/income-tax/adjustable-summary-calculation/${taxYear.asTysDownstream}/${nino.nino}",
+        body = tysModel
+      ).returns(Future.successful(outcome))
+
+      await(connector.triggerBsas(request)) shouldBe outcome
+    }
+
```

Comment:
Should probably add some negative tests too

i.e. `
"return an unsuccessful response" should {
      "a non-valid request with a single error be supplied" in new IfsTest with Test {...}
      "a non-valid request with multiple errors be supplied" in new IfsTest with Test {...}
}
`